pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -145,6 +145,10 @@ self: super: {
     then addBuildDepend super.halive pkgs.darwin.apple_sdk.frameworks.AppKit
     else super.halive;
 
+  barbly = if pkgs.stdenv.isDarwin
+    then addBuildDepend super.barbly pkgs.darwin.apple_sdk.frameworks.AppKit
+    else super.barbly;
```

Comment:
The change to `configuration-hackage2nix.yaml` suggests that `barbly` is not supported on platforms other than Darwin. Therefore, I believe the if-then-else conditional to be unnecessary here. If `barbly` is built, then it's necessarily for a Darwin target, so just add that framework unconditionally, IMHO.